### ISSUE TITLE
Keyboard focus is not logical in chart section on the page 

### ISSUE THREAD

**Issue by anubhutisaini12** [highcharts/highcharts#14821]
#### Expected 
Keyboard tab focus should not moves two times on same control that appears under the chart section of the page also focus should not on hidden element after each chart section navigation. 
Hence keyboard focus should be in logical order on the page. 

#### Actual behavior
Keyboard tab focus moving two times on same control that appears under the chart section of the page also focus moving on hidden element after each chart section navigation. keyboard focus again move to left to right order over the graph section.
Hence keyboard focus is not in logical order on the page. 

#### Live demo with steps to reproduce

https://www.highcharts.com/demo/line-basic

![image](https://user-images.githubusercontent.com/76141946/102440820-f9eed780-4046-11eb-8e2b-0755a46c31c7.png)

#### Product version
 Highcharts 2.2.2
#### Affected browser(s)
Edge

**Comment by KacperMadej** [comment]
Hi @anubhutisaini12 ,

Thank you for reporting the problem.

I am unable to recreate the problem you have described. Please add more details and info about the used browser and OS.

On Windows10/Chrome after click to focus on the light grey box around the chart the first 5 tab clicks are as shown in the image below:

![tab-through-demo](https://user-images.githubusercontent.com/8960408/102507591-ba46e080-4084-11eb-97bf-a024c8a7082f.jpg)

(1 - go to the previous chart, 2 - chart's point, 3 - chart's legend, 4 - chart's menu, 5 - go to the next chart)

**Comment by anubhutisaini12** [comment]
When we use accessibility Insight tool (chrome extension) then focus moves to the footer of the chart.

![image](https://user-images.githubusercontent.com/76141946/107469995-bb7e2080-6b90-11eb-92e4-7fea3a9fe3ef.png)

**Comment by KacperMadej** [comment]
I have tested the extension available here: https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni

It looks like the tool you are using is not working perfectly when it comes to interactive SVG elements that trigger JS code. This is a problem within the tool as it fails to mark correctly the first hovered point (should be No.9), first legend item (No.10) and jumps after a single tab by 2 items (over the problematic No.12 and right to the No.13). After the main chart div a HTML elements for a11y are added and those might be pointed by the tool, but I this would also be an error of the tool.

fyi (a11y) @oysteinmoseng 

_Internal note:_
The tool reports a11y failed contrast for 2 elements in the image in the previous comment - should this be investigated as a part of a new issue for `Produt: Website` ?  @oysteinmoseng

**Comment by oysteinmoseng** [comment]
Yes this looks like an issue with Accessibility Insights - closing. Feel free to comment and we can reopen if needed @anubhutisaini12. 

@KacperMadej Yes, good call! I will make a new issue.
